pkg/driver/microphone/microphone.go

```diff
@@ -111,6 +111,8 @@ func (m *microphone) AudioRecord(inputProp prop.Media) (audio.Reader, error) {
 	config.PerformanceProfile = malgo.LowLatency
 	config.Capture.Channels = uint32(inputProp.ChannelCount)
 	config.SampleRate = uint32(inputProp.SampleRate)
+	//FIX: Turn on the microphone with the current device id
```

Comment:
As it seems already fixed by this PR, removing `FIX:` would be less confusing.

Maybe
```suggestion
	// Select the microphone with the current device id
```
?